cli.js

```diff
@@ -75,11 +75,38 @@ program
 			'--sample',
 			'Get a sample of proxies from each source'
 		)
+		.option(
+			'--stdout',
+			'Write to STDOUT instead of a file'
+		)
 		.action(function() {
 
 			var outputFormat = this.outputFormat;
+			var stdout = this.stdout;
 			var outputFile = process.cwd() + '/' + this.outputFile + '.' + this.outputFormat;
-			var writeStream = fs.createWriteStream(outputFile);
+			var writeStream;
+
+			if(!stdout) {
+				writeStream = fs.createWriteStream(outputFile);
+			}
+			else {
+				outputFile = 'STDOUT';
+				var logFile = process.cwd() + '/proxy-lists.log';
+				var logStream = fs.createWriteStream(logFile);
+				writeStream = {
+					write: function(data) {
+						process.stdout.write(data + '\n');
+					},
+					end: function() {},
+					on: function() {}
+				};
+
+				// Pipe console.log output to the log file
+				console.log = function(message) {
```

Comment:
Not necessarily. Could create a new function that writes to either console.log or the log file based on the current options.